src/store/actions/contact.actions.ts

```diff
@@ -97,11 +97,12 @@ const getUserContact = async (): Promise<IContact> => {
     name: userFullName,
     alias: userFullName,
     uri: user.emailAddress,
-    roles: [IdentityRoleEnum.ISSUER],
+    //todo: handle this based on the identities available in the wallet
```

Comment:
would be nice to make a ticket for this, so that if we have time we can implement this for v1

Reply:
created a ticket and updated